src/selectors/entities/general.ts

```diff
@@ -37,6 +37,10 @@ export function warnMetricsStatus(state: GlobalState): any {
     return state.entities.general.warnMetricsStatus;
 }
 
+export function subscriptionStatsFromState(state: GlobalState): any {
+    return state.entities.cloud.subscriptionStats;
+}
+
```

Comment:
I think the name getSubscriptionStats is more in-line with the rest of the selectors in this file, and I think it highlights that this is a selector a little better at a glance

Reply:
Thanks @nickmisasi ,ok, I will change it then!